DOC-1724: Update ToC and EditImage minimum plans (#2456)
tinymce/tinymce-docs: modules/ROOT/pages/tableofcontents.adoc

:description: Insert a simple Table of Contents into the {productname} editor
:keywords: tableofcontents, tableofcontents_depth, tableofcontents_class, tableofcontents_header, toc
:pluginname: Table of Contents

> :pluginnamelower: table of contents

Reviewer: Should this really be a variable? This is not a plugin name it's just plain text...

- [ ] Remove variable here

Author: I was thinking there were more cases but only found one, fixing now

:plugincode: tableofcontents
:pluginminimumplan: tiertwo

include::partial$misc/admon-premium-plugin.adoc[]

> The {pluginname} plugin will generate basic {pluginnamelower} and insert it into the editor at the current cursor position. Items for the table will be taken from the headers found in the content.

Reviewer 2: ```suggestion
The {pluginname} plugin generates a basic {pluginnamelower} (ToC) and inserts it into the editor at the current cursor position. ToC entries are generated from header elements (`h1 – h7` elements) in the content.
```

Minor edits. I resisted the temptation to `s/cursor position/insertion point/`. The I-beam pointer is not a cursor. But that’s an editorial change that I need to argue for and then do across the entire documentation suite.

Reviewer: `h7` doesn't exist... I'm guessing that was a typo and was meant to be `h6`?

> NOTE: In previous versions of {productname}, the {pluginname} plugin was provided as an open source plugin.

Reviewer 2: ```suggestion
NOTE: in previous versions of {productname} the {pluginname} plugin was provided as an open source plugin also called {pluginname}.
```

The edit above assumes I am right about the previous open source plugin’s name.

Author: That is correct

Reviewer: `In` should not be lowercased though as it's the start of the Paragraph since `NOTE:` is markup here.

Reviewer: Unresolving as my note above here was missed...

:keywords: tableofcontents, tableofcontents_depth, tableofcontents_class, tableofcontents_header, toc
:pluginname: Table of Contents
:plugincode: tableofcontents
:pluginminimumplan: tiertwo

include::partial$misc/admon-premium-plugin.adoc[]

> The {pluginname} plugin generates a basic _table of contents_ (ToC) and inserts it into the editor at the current cursor position. ToC entries are generated from header elements (`h1 – h7` elements) in the content.

Reviewer 2: ```suggestion
The {pluginname} plugin generates a basic _table of contents_ (ToC) and inserts it into the editor at the current cursor position. ToC entries are generated from header elements (`h1 – h6` elements) in the content.
```
Typo correction: `s/h7/h6`.

> NOTE: in previous versions of {productname} the {pluginname} plugin was provided as an open source plugin also called {pluginname}.

Reviewer 2: ```suggestion
NOTE: In previous versions of {productname} the {pluginname} plugin was provided as an open source plugin also called {pluginname}.
```
